spidermon/contrib/scrapy/pipelines.py

```diff
@@ -41,6 +46,10 @@ def __init__(
 
     @classmethod
     def from_crawler(cls, crawler):
+        spidermon_enabled = crawler.settings.getbool("SPIDERMON_ENABLED")
+        if not spidermon_enabled:
```

Comment:
Would it disable for those projects where we have not set SPIDERMON_ENABLED flag explicitly ?

Reply:
Yes, it will. I am considering that item validation pipeline is a spidermon related pipeline, so if we don't explicitly enable spidermon, everything spidermon related should not be executed.

We had this problem in a project where spidermon was disabled in production but the pipeline was there with SPIDERMON_VALIDATION_ADD_ERRORS_TO_ITEMS set to True. This leads to production items with `_validation` field populated which is not desired.

Reply:
The question I get here is, does it make sense to have only monitoring or validation enabled?
Say, is there a case where I just monitoring, but not validation?
Or, is there a case I just want validation, but no monitoring?
If so, I'd go for two setting instead of one.
Otherwise, if we consider spidermon as a whole, then this is the preferred solution

Reply:
Yes, it does make sense to use just the validation feature of Spidermon or just the monitoring part. However in this case I am considering Spidermon as a whole, so we should be able to enable/disable everything Spidermon related using just one setting to avoid problems like the one I mentioned.